src/platform/Infineon/PSOC6/ConnectivityManagerImpl.cpp

```diff
@@ -176,12 +176,13 @@ CHIP_ERROR ConnectivityManagerImpl::_GetAndLogWiFiStatsCounters(void)
 {
     cy_wcm_associated_ap_info_t ap_info;
     cy_rslt_t result = CY_RSLT_SUCCESS;
+    CHIP_ERROR err   = CHIP_NO_ERROR;
 
     result = cy_wcm_get_associated_ap_info(&ap_info);
     if (result != CY_RSLT_SUCCESS)
     {
         ChipLogError(DeviceLayer, "cy_wcm_get_associated_ap_info failed: %d", (int) result);
-        SuccessOrExit(CHIP_ERROR_INTERNAL);
+        SuccessOrExit(err = CHIP_ERROR_INTERNAL);
```

Comment:
We should add a lint for `SuccessOrExit(CHIP_ERROR` to `.github/workflows/lint.yml`.  Followup is fine for that.

Reply:
Created https://github.com/project-chip/connectedhomeip/pull/26057